src/v/kafka/server/handlers/describe_configs.cc

```diff
@@ -235,15 +329,19 @@ static void add_topic_config_if_requested(
   const std::optional<T>& default_value,
   std::string_view override_name,
   const tristate<T>& overrides,
-  bool include_synonyms) {
+  bool include_synonyms,
+  bool include_documentation,
```

Comment:
maybe a better API would be to have documentation be an optional. Seems a little weird that if false is passed, then the next parameter will be unused. The caller has all of that information before hand.